book/website/project-design/sdp/types.md

```diff
@@ -5,6 +5,28 @@ Sensitive data is information that must be protected against unauthorised access
 Protection of this data may therefore be required for legal or ethical reasons.
 
+When measuring how sensitive a data is, different organizations may follow (or are required to follow) certain structure in identifying different levels of sensitive data and information, based on frameworks. In general, the sensitivity levels are broadly classified into three categories:
+* **High sensitivity / highly restricted data** - It could include details like sensitive personal data, salary information, bank details, research data with significant commercial value or obligations. If this kind of data is compromised or destroyed in an unauthorized way, it could greatly impact the organizations/individuals involved and they could face repercussions. Unauthorized disclosure may result in fines, legal action, reputational damage, economic losses, and other consequences.
+* **Medium sensitivity / restricted data** - It could include details like personal data that is not classified as 'sensitive', student/alumni contact details, staff contact details. This kind of data is usually for internal usages only, and if compromised or destroyed in an unauthorized way, would not be as impactful as the former on the organizations/individuals involved.
+* **Low sensitivity / internal data** - It could include details like non-confidential internal correspondence, working group minutes, internal policies and procedures. This kind of data could also comprise of data intended for public use, like public website contents or press releases.
+
+According to the information guidelines provided by [Imperva](https://www.imperva.com/learn/data-security/data-classification/), data could be classified based on content, context or user-based. 
+* **Content-based classification** - This type of classification would involve inspecting and interpreting files and documents looking for sensitive information while reviewing them, and classifying based on that.
+* **Context-based classification** - This type of classification would involve looking into the metadata to classify, such as the application which created the file, person/people involved in its creation, or the location of its authorization. Some questions one can ask while looking for this classification are:
+  * How is the data being used? 
+  * Who has the access to it? 
+  * When are they accessing it? 
+* **User-based classification** - This type of classification would involve the expertise of judgement made by one or more knowledgeable users in classifying the file or document. Hence, they would be advising on specifying how sensitive the data is and make necessary changes or edits before they release it.
+
+In terms of research, as per the detailed guidelines provided by various universities across the UK (and elsewhere) abiding by the Data Protection Act, the most common reasons data falls under the high sensitivity data are:
```

Comment:
```suggestion
In terms of research the most common reasons data falls under the high sensitivity data are:
```
You already mention the laws/frameworks above, so I don't think its needed to repeat that info here.